controllers/argocd/notifications/constants.go

```diff
@@ -0,0 +1,16 @@
+package notifications
+
+// Values
+const (
+	// ArgoCDNotificationsControllerComponent is the name of the Notifications controller control plane component
+	ArgoCDNotificationsControllerComponent = "notifications-controller"
+	RoleKind                               = "Role"
+	TLSCerts                               = "tls-certs"
+	ArgoCDRepoServerTLS                    = "argocd-repo-server-tls"
+	CapabilityDropAll                      = "ALL"
+	VolumeMountPathTLS                     = "/app/config/tls"
+	VolumeMountPathRepoServerTLS           = "/app/config/reposerver/tls"
+	WorkingDirApp                          = "/app"
+	ImageUpgradedLabel                     = "image.upgraded"
+	TimeFormatMST                          = "01022006-150406-MST"
```

Comment:
most of these constants are not notifications related, and are more general purpose. They should not be defined in the notifications package

I will raise a separate PR to put them in a common location, which can be referenced